doot-doot/cogs/Airhorn.py

```diff
@@ -228,6 +228,11 @@ def __init__(self, bot):
         """Honk Honk!"""
         await play_file(ctx, "sounds/honk.mp3")
 
+   @commands.command()
+    @commands.guild_only()
+    async def nofreakingway(self, ctx):
+        """From the Megamind (2010) movie soundboard."""
+        await play_file(ctx, "sounds/no-freaking-way!.mp3")
```

Comment:
for safety please remove ! from file, it technically shouldnt be an issue but i want to play it safe

Reply:
The ! was in the mp3 name, so there shouldn't be an issue since it's the exact name of the file. I can reupload without the ! if needed, though.